stuff/search-handler.go

```diff
@@ -158,23 +157,23 @@ func (h *SearchHandler) apiSearchPageItem(out http.ResponseWriter, r *http.Reque
 		Items:      make([]JsonHtmlSearchResultRecord, outlen),
 	}
 
-	pusher, _ := out.(http.Pusher) // HTTP/2 pushing if available.
+//	pusher, _ := out.(http.Pusher) // HTTP/2 pushing if available.
```

Comment:
Please don't include this in the pull request for upstream.
You probably need to comment that out in a branch locally if you run an old golang.